packages/stream-model-instance/src/model-instance-document.ts

```diff
@@ -85,37 +61,20 @@ function headerFromMetadata(
   metadata: ModelInstanceDocumentMetadataArgs | StreamMetadata | undefined,
   genesis: boolean
 ): CommitHeader {
-  if (typeof metadata?.schema === 'string') {
-    try {
-      CommitID.fromString(metadata.schema)
-    } catch {
-      throw new Error('Schema must be a CommitID')
-    }
-  }
 
   const header: CommitHeader = {
     controllers: metadata?.controllers,
-    family: metadata?.family,
-    schema: metadata?.schema?.toString(),
-    tags: metadata?.tags,
   }
 
   // Handle properties that can only be set on the genesis commit.
   if (genesis) {
-    if (!metadata?.deterministic) {
-      header.unique = uint8arrays.toString(randomBytes(12), 'base64')
```

Comment:
I think we still need to include the `unique` random bytes by default, e.g. there should be no way to disable it.

Reply:
Gotcha, yeah missed that. Added back in.